chia/simulator/full_node_simulator.py

```diff
@@ -444,7 +445,7 @@ async def process_coin_spends(self, coins: Collection[Coin] = ()) -> None:
         block.
 
         Arguments:
-            coin_names: The coin names to process.
+            coins: The coins to process.
```

Comment:
this is unrelated, and not obvious that it's an improvement

Reply:
Will revert.

Reply:
This is unrelated, but notice that this is changing the documentation to match the actual parameter name and type. I'll break this out into a separate PR.

Reply:
Done.

Reply:
@arvidn This was moved here: https://github.com/Chia-Network/chia-blockchain/pull/14112/files